src/styles/card.css

```diff
@@ -0,0 +1,147 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: fit-content;
+  padding: 16px;
+
+  background: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --color-navy-blue: #060b35;
+  --color-blue: #00acdc;
+  --color-light-gray: #616070;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 16px auto 0;
+}
+
+.card__text {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin: 0;
+}
+
+.card__text--1 {
+  width: 166px;
+  height: 36px;
```

Comment:
@mmahfuzi this style is applied to `<p` element - does it actually impact the layout?
nothing changes if you just remove it.